.github/workflows/nightly.yml

```diff
@@ -88,7 +89,7 @@ jobs:
   build_linux:
     name: 'Linux build'
     runs-on: ubuntu-latest
-
+    
```

Comment:
Please remove orphaned spaces.

Reply:
YML... ugh. Sorry about that!
Should be good to go.